monitor/monitoring-dam-generator-core/src/main/java/eu/seaclouds/monitor/monitoringdamgenerator/dcgenerators/MODACloudsDcGenerator.java

```diff
@@ -17,10 +15,10 @@
             .getLogger(MODACloudsDcGenerator.class);
 
     private static final String MODACLOUDS_DC_ID = "modacloudsDc";
-    private static final String START_SCRIPT_URL = "http://start.sh/";
+    private static final String START_SCRIPT_URL = "http://installMODACloudsDc.sh/";
```

Comment:
Is the current solution I described before compliant with the first one you propose or it needs to be implemented differently?